src/content/docs/en/reference/configuration-reference.mdx

```diff
@@ -1161,7 +1163,7 @@ and deduplicating scripts). You can also now conditionally render scripts in you
 However, this means scripts are no longer hoisted to the `<head>` and multiple scripts on a page are no longer bundled together.
 If you enable this option, you should check that all your `<script>` tags behave as expected.
 
-This option will be enabled by default in Astro 5.0.
+This option will be enabled by default in Astro 4.5.0.
```

Comment:
How about this part?

If the PR for this part is correct, I would like to open a new PR.